lib/quality_viewer/subscriptions/subscription.ex

```diff
@@ -0,0 +1,18 @@
+defmodule QualityViewer.Subscriptions.Subscription do
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  schema "subscriptions" do
+    belongs_to :subscriber, QualityViewer.Accounts.User
+    belongs_to :subscribed_to, QualityViewer.Accounts.User
+    field :currently_subscribed, :boolean
```

Comment:
Actually, we'd instead remove the record when they do unsubscribe.
So the presence of the row would mean that they're subscribed